packages/protocol/test/layer2/LibEIP1559.t.sol

```diff
@@ -98,23 +98,23 @@ contract TestLibEIP1559 is Layer2Test {
         assertEq(baselineBasefee, basefee);
     }
 
-    /// forge-config: layer2.fuzz.runs = 1000
-    /// forge-config: layer2.fuzz.show-logs = true
+    /// forge-config: layer_2.fuzz.runs = 1000
```

Comment:
why change to "layer_2"?

Reply:
forge-config will convert layer2 to layer_2 in *.t.sol files when fuzz.runs is used.

Reply:
Updated to `forge-config: default.fuzz_runs = 2000`